conda/common/logic.py

```diff
@@ -785,20 +807,20 @@ def BDD(self, equation, nterms, lo, hi, polarity):
             ret[call_stack_pop()] = ITE(abs(LA), thi, tlo, polarity, add_new_clauses=True)
         return ret[target]
 
-    def LinearBound(self, equation, lo, hi, preprocess, polarity):
+    def LinearBound(self, lits, coeffs, lo, hi, preprocess, polarity):
```

Comment:
We do not check if `len(lits) == len(coeffs)` since the inputs from `Clauses`' `LinearBound` and `minimize` methods come from `dict.keys` + `dict.items`, hence are equal in length.
We could add checks for the length here if we want, though.